backend/api/office_hours/oh_section.py

```diff
@@ -0,0 +1,92 @@
+"""OH Section API
+
+This API is used to access OH section data."""
+
+from fastapi import APIRouter, Depends
+
+from backend.models.office_hours.oh_section import OfficeHoursSectionDraft
+from backend.models.office_hours.oh_section_details import OfficeHoursSectionDetails
+from ..authentication import registered_user
+from ...models import User
+
+
+__authors__ = ["Sadie Amato", "Bailey DeSouza"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+api = APIRouter(prefix="/api/office-hours/section")
+openapi_tags = {
+    "name": "Office Hours",
+    "description": "Office hours section functionality",
+}
+
```

Comment:
Similar to Office Hours Events, do you think we should have an endpoint to get an office hours section by `id`? Since we have established relationships in the Office Hours Section model/details, having this model could be helpful to display the related in the frontend!

Reply:
I've added this in the recent commits, thank you!